docs/pages/access-controls/sso/github-sso.mdx

```diff
@@ -32,86 +32,110 @@ https://PROXY_ADDRESS/v1/webapi/github/
 address of the Teleport Proxy Service</ScopedBlock><ScopedBlock
 scope="cloud">your Teleport Cloud tenant address</ScopedBlock>.
 
-Instructions for creating a GitHub OAuth app are available here:
+Instructions for creating a GitHub OAuth app are available in [GitHub's
+documentation](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app)
 
-[Creating an OAuth App](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app)
+Create a client secret to use along with the client ID in the next step.
 
 ## Step 2/3. Create a GitHub authentication connector
 
-Define a GitHub authentication connector by creating a file called `github.yaml`
-with the following content:
+Define a GitHub authentication connector using `tctl`. Update this example
+command with your OAuth app's client ID, client secret, and the roles you want
+to map from your GitHub organization to Teleport roles. See [tctl sso configure
+github](../../reference/cli.mdx#tctl-sso-configure-github) for a full reference
+of flags for this command:
```

Comment:
We say that we need client id, client secret, org name and team name, but if someone isn't well versed in the lingo, they will be lost. Perhaps we could present a short table explaining the necessary tokens, what they mean, and where they come from.

Reply:
Resolved in 0c4879515c00988313f04af6bd9795e81b99d987